test/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/SubsidyControllerSpec.scala

```diff
@@ -1772,7 +1772,7 @@ class SubsidyControllerSpec
           checkFormErrorIsDisplayed(
             performAction()("TID1234"),
             messageFromMessageKey("subsidy.remove.title"),
```

Comment:
I know you didn't add this line but can you update it as well so it doesn't load from messages